src/app/rpmostree-dbus-helpers.c

```diff
@@ -809,6 +807,32 @@ vardict_insert_strv (GVariantDict *dict,
 }
 
 static gboolean
+vardict_sort_and_insert_pkgs (GVariantDict *dict,
```

Comment:
Don't need to change this PR, but I've been trying to add at least single-line function comments as I go along.